### ISSUE TITLE
/staff endpoint broken

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispersservices_django#322]
on dev and test, `whispersservices/staff` returns this error:

![image](https://user-images.githubusercontent.com/6978380/69740706-6c6cdc80-10ff-11ea-98aa-253bcf20c39e.png)

On prod it responds with a server-level "The specified URL cannot be found." error.

Traceback:
```
Traceback:

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/fields.py" in get_attribute
  454.             return get_attribute(instance, self.source_attrs)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/fields.py" in get_attribute
  102.                 instance = getattr(instance, attr)

During handling of the above exception ('Comment' object has no attribute 'first_name'), another exception occurred:

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  126.                 response = self.process_exception_by_middleware(e, request)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  124.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/viewsets.py" in view
  116.             return self.dispatch(request, *args, **kwargs)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/views.py" in dispatch
  495.             response = self.handle_exception(exc)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/views.py" in handle_exception
  455.             self.raise_uncaught_exception(exc)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/views.py" in dispatch
  492.             response = handler(request, *args, **kwargs)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/mixins.py" in list
  45.             return self.get_paginated_response(serializer.data)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/serializers.py" in data
  765.         ret = super(ListSerializer, self).data

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/serializers.py" in data
  262.                 self._data = self.to_representation(self.instance)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/serializers.py" in to_representation
  683.             self.child.to_representation(item) for item in iterable

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/serializers.py" in <listcomp>
  683.             self.child.to_representation(item) for item in iterable

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/serializers.py" in to_representation
  514.                 attribute = field.get_attribute(instance)

File "/www/htdocs/test/whispersservices_django/env/lib64/python3.6/site-packages/rest_framework/fields.py" in get_attribute
  475.             raise type(exc)(msg)

Exception Type: AttributeError at /staff/
Exception Value: Got AttributeError when attempting to get a value for field `first_name` on serializer `StaffSerializer`.
The serializer field might be named incorrectly and not match any attribute or key on the `Comment` instance.
Original exception text was: 'Comment' object has no attribute 'first_name'.
```

**Comment by BlakeDraper** [comment]
Related: https://github.com/USGS-WiM/whispers/issues/751

**Comment by aaronstephenson** [comment]
Fixed in 1.4.2 d6d58ca
